java/src/jmri/util/swing/JMenuUtil.java

```diff
@@ -192,5 +196,50 @@ static int convertStringToKeyEvent(@Nonnull String st) {
         return kcode;
     }
 
+    /**
+     * replace a menu item in its parent with another menu item
+     * <p>
+     * (at the same position in the parent menu)
+     *
+     * @param orginalMenuItem the original menu item to be replaced
+     * @param replacementMenuItem the menu item to replace it with
+     * @return true if the original menu item was found and removed or replaced
+     */
+    public static boolean replaceMenuItem(
+            @Nullable JMenuItem orginalMenuItem,
```

Comment:
Because old habits die hard and I tend to write paranoid code (that checks all it's inputs). I do see an advantage of being able to pass null pointers and having that tested once internally vs. having to always test them externally before calling.
That said, I'm not married to passing it @Nullable if you think I shouldn't.

Reply:
Ok, done!